src/main/java/com/amazonaws/kinesisvideo/producer/KvsRetryStrategyType.java

```diff
@@ -0,0 +1,14 @@
+package com.amazonaws.kinesisvideo.producer;
+
+public enum KvsRetryStrategyType {  
```

Comment:
suggest add a link to the native for refrence..

Reply:
Done